app/renderer/routes.tsx

```diff
@@ -1,41 +1,71 @@
-import { SignupPage } from "app/renderer/ui/pages/signup"
 import * as React from "react"
-import { Route } from "react-router"
 import { Store } from "redux"
+import { Route, Switch } from "react-router"
 
-import App from "app/renderer/ui/containers/App"
-import { MainPage } from "app/renderer/ui/pages/main"
 import { StoreState } from "app/renderer/store"
-
-import { RedirectedRoute } from "app/renderer/utils/redirectedRoute"
-import { ifWallet, ifWallets } from "app/renderer/utils/guards"
-import LoginFormContainer from "app/renderer/ui/containers/LoginForm"
+import App from "app/renderer/ui/containers/App"
+import {
+  default as LoginFormContainer,
+  PATHS as PATHS_LOGIN_FORM
+} from "app/renderer/ui/containers/LoginForm"
+import MainPage from "./ui/pages/main"
+import { SignupPage } from "./ui/pages/signup"
+import { ifWallets } from "./utils/guards"
+import { RedirectedRoute } from "./utils/redirectedRoute"
 
 type RoutesProps = {
   store: Store<StoreState>
 }
 
 export default (props: RoutesProps) => {
-  const { store } = props
 
   return (
     <App>
-      <RedirectedRoute
-        path="/"
-        guard={ifWallet(store)}
-        component={MainPage}
-        redirect="/login"
-      />
-      <RedirectedRoute
-        path="/login"
-        guard={ifWallets(store)}
-        component={LoginFormContainer}
-        redirect="/wallets/new"
-      />
-      <Route
-        path="/wallets/new"
-        component={SignupPage}
-      />
+      <Switch>
+        <Route
+          // TODO: update with PATHS from new wallet form container
+          path="/forms/wallet"
+          render={newWalletRenderer}
+        />
+        <Route
+          // TODO: update with PATHS from Main container
+          path="/main"
+          render={mainRenderer}
+        />
+        <Route
+          path={PATHS_LOGIN_FORM.ROOT}
```

Comment:
This forces the router inside the login form to define a default route pointing to its own welcome screen. As we discussed, we could drop default routes and make all routes explicit altogether.